_topic_maps/_topic_map_ms.yml

```diff
@@ -51,7 +51,7 @@ Distros: microshift
 Topics:
 - Name: Installing from an RPM package
   File: microshift-install-rpm
-- Name: Installing for FIPS compliance
```

Comment:
There is a problem here. The title in the map is "Installing for FIPS mode" but the title in the assembly is "Running {microshift-short} containers in FIPS mode". This creates a mismatch for the user - the navigation is one title, but when clicked, another title opens

Reply:
Also the content does not seem to have a proper procedure despite gerunds in the titles, though this is probably beyond the scope of the current change. At least please fix the title mismatch

Reply:
The procedure is based on RHEL, yeah, it is not really a MicroShift procedure per se. So noted. TY